exporters/otlp/include/opentelemetry/exporters/otlp/recordable.h

```diff
@@ -21,7 +21,13 @@ class Recordable final : public sdk::trace::Recordable
   void SetAttribute(nostd::string_view key,
                     const opentelemetry::common::AttributeValue &value) noexcept override;
 
-  void AddEvent(nostd::string_view name, core::SystemTimestamp timestamp) noexcept override;
+  void AddEvent(nostd::string_view name,
+                core::SystemTimestamp timestamp,
+                const trace::KeyValueIterable &attributes) noexcept override;
```

Comment:
Do we want to provide an overload which allows just name and timestamp?

The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#add-events) mentioned that timestamp and attributes are optional.

Reply:
`Span::AddEvent` supports all combinations (set name only, set name and timestamp, and set name, timestamp and attributes). If `Span` doesn't want to use some of these parameters it can set them to empty. This will reduce the number of functions we have to maintain in the `Recordable`. What do you think? I don't have a strong preference either way.

Reply:
No strong preference. I guess we probably can have the default value for timestamp and attributes?

Reply:
And my suggestion was from the simplicity perspective, we don't want to force people to write more code.

Reply:
Yes, makes sense! Should I add default timestamp and attribute values to the function signature?

Reply:
I would request a default attributes at minimum. A default timestamp is nice to have. 😄

Reply:
I'm fine with both having the current time and an empty `KeyValueIterable` as default argument or passing those values in the SDK implementation of `Span`.

Reply:
I've added default parameters for timestamp and attributes. I've also added a default parameter for link attributes.